qemu/tests/bridge_mirror.py

```diff
@@ -77,28 +77,34 @@ def check_tcpdump(output, src_ip, des_ip, ping_count):
                 reply_num += 1
 
         if request_num != ping_count or reply_num != ping_count:
-            logging.debug(
-                "Unexpect request or reply number. "
-                "current request number is: %d, "
-                "current reply number is: %d, "
-                "expected request and reply number is: %d. "
-                % (request_num, reply_num, ping_count))
+            logging.debug("Unexpect request or reply number. "
+                          "current request number is: %d, "
+                          "current reply number is: %d, "
+                          "expected request and reply number is: %d. "
+                          % (request_num, reply_num, ping_count))
             return False
         return True
 
+    netdst = params.get("netdst", "switch")
```

Comment:
Hi, these four lines seems redundant since you already add `only bridge` in cfg file, please delete this check.

Reply:
`only bridge` contains `linux bridge` and `ovs bridge`, these bridge testcases just support `linux bridge`, so need them to skip `ovs bridge` backend.

Reply:
@sitoliu Name it like that will mislead user, since `linux_` looks like something related to guest os. Let's take `ovs` a specific type of `bridge`. Only exclude ovs type like current implementation in the code is fine.

Reply:
And we have another testcase named `ovs_mirror`, it tests the port mirroring with ovs backend.
So we can use `ovs_` and `bridge_` to distinguish between 'ovs bridge' and 'linux bridge'

Reply:
If you want to use `bridge_` as prefix name for `linux bridge only` case, I suggest to add some comment in these files to distinguish it.

Reply:
I updated as follows:
``` python
def run(test, params, env):
    """
    Test 802.1Q vlan of NIC among guests and host with linux bridge backend.
```
``` python
def run(test, params, env):
    """
    Test port mirror with linux bridge backend.
```